index.html

```diff
@@ -37,8 +38,199 @@ <h1>I'm Joseph Glad to meet you!</h1>
+        
+    </section>
+    <section class="grid-item">
+        <img class="snap" src="Images/Snapshoot Portfolio (2).png" alt="">
+        <h2 class="multi">Multi-Post
+            Stories</h2>
+        <div class="container">
+            <div class="canopy">
+                <p>CANOPY</p>
+            </div>
+            
+            <div class="details">
+                <img src="Images/Counter.png" alt="">
+                <p>Backend Dev</p>
+            </div>
+            <div class="details">
+                <img src="Images/Counter.png" alt="">
+                <p>2015</p>
+            </div>
+        </div>
+        <div class="content">
+            <p>
+                A daily selection of privately personalized reads; 
+                no accounts or sign-ups required.
+            </p>
+        </div>
+        
+         <ul class="link">
+            <li><a href="">Html</a></li>
+            <li><a href="">css</a></li>
+            <li><a href="">javascript</a></li>
+         </ul>
+
+         <button class="button"><a href="">See Project</a></button>
+        
+    </section>
+    <section class="grid-item">
+        <img class="snap" src="Images/Snapshoot Portfolio (1).png" alt="">
+        <h2 class="tonic">Tonic</h2>
+        <div class="container">
+            <div class="canopy">
+                <p>CANOPY</p>
+            </div>
+            
+            <div class="details">
+                <img src="Images/Counter.png" alt="">
+                <p>Backend Dev</p>
+            </div>
+            <div class="details">
+                <img src="Images/Counter.png" alt="">
+                <p>2015</p>
+            </div>
+        </div>
+        <div class="content">
+            <p>
+                A daily selection of privately personalized reads; 
+                no accounts or sign-ups required.
+            </p>
+        </div>
+        
+        <ul class="link">
+            <li><a href="">Html</a></li>
+            <li><a href="">css</a></li>
+            <li><a href="">javascript</a></li>
+         </ul>
+
+         <button class="button"><a href="">See Project</a></button>
+        
+    </section>
+    <section class="grid-item">
+        <img class="snap" src="Images/Snapshoot Portfolio (3).png" alt="">
+        <h2 class="multi">Multi-Post
+            Stories</h2>
+        <div class="container">
+            <div class="canopy">
+                <p>CANOPY</p>
+            </div>
+            
+            <div class="details">
+                <img src="Images/Counter.png" alt="">
+                <p>Backend Dev</p>
+            </div>
+            <div class="details">
+                <img src="Images/Counter.png" alt="">
+                <p>2015</p>
+            </div>
+        </div>
+        <div class="content">
+            <p>
+                A daily selection of privately personalized reads; 
+                no accounts or sign-ups required.
+            </p>
+        </div>
```

Comment:
As requested by the previous reviewer, Kindly make sure the work section is centered as per the Figma design:

Your design:
![Screenshot from 2023-02-18 21-13-53](https://user-images.githubusercontent.com/97834160/219884407-56e8e105-9b1a-47f0-a9a5-b44c294061bd.png)